lib/iex/lib/iex/helpers.ex

```diff
@@ -394,6 +394,11 @@ defmodule IEx.Helpers do
 
   @doc """
   Prints information about the given data type.
+  
+  ## Examples
+
+      i(Int)
+      i(Atom)
```

Comment:
I can see what you're getting at. And it helped me a lot. I didn't know I could use i to figure out the type of any valid Elixir term. The docs suggest that you give it a data type. I'll make a another commit.